### PR TITLE
docs: Update FAQ replace momentjs to Day.js with antd-dayjs-webpack-plugin

### PR DESCRIPTION
**Pull request by iamkun**
### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [x] Other (about what?) FAQ update

### 🔗 Related issue link
https://github.com/ant-design/ant-design/issues/19738

### 💡 Background and solution

A webpack plugin [antd-dayjs-webpack-plugin](https://github.com/ant-design/antd-dayjs-webpack-plugin) to replace momentjs to Day.js.

More info at https://github.com/ant-design/ant-design/issues/19738

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

-----
[View rendered docs/react/faq.en-US.md](https://github.com/iamkun/ant-design/blob/doc/dayjs-webpack-plugin/docs/react/faq.en-US.md)
[View rendered docs/react/faq.zh-CN.md](https://github.com/iamkun/ant-design/blob/doc/dayjs-webpack-plugin/docs/react/faq.zh-CN.md)

### CODE REVIEW

**Comment by afc163** [comment]
可以加到 https://ant.design/docs/react/getting-started-cn#%E6%8C%89%E9%9C%80%E5%8A%A0%E8%BD%BD ，『按需加载』后面来一个段落具体说一下怎么操作。

https://ant.design/docs/react/use-with-create-react-app-cn#%E8%87%AA%E5%AE%9A%E4%B9%89%E4%B8%BB%E9%A2%98 『自定义主题』后也加上。

**Comment by iamkun** [comment]
> 可以加到 https://ant.design/docs/react/getting-started-cn#%E6%8C%89%E9%9C%80%E5%8A%A0%E8%BD%BD ，『按需加载』后面来一个段落具体说一下怎么操作。
> 
> https://ant.design/docs/react/use-with-create-react-app-cn#%E8%87%AA%E5%AE%9A%E4%B9%89%E4%B8%BB%E9%A2%98 『自定义主题』后也加上。

恩 好的

File: docs/react/use-with-create-react-app.zh-CN.md [hunk 1]
```diff
@@ -201,6 +201,21 @@ module.exports = override(
 
 > 你也可以使用 [craco](https://github.com/sharegate/craco) 和 [craco-antd](https://github.com/FormAPI/craco-antd) 来实现和 customize-cra 一样的修改 create-react-app 配置的功能。
 
+## 使用 Day.js 替换 momentjs 优化打包大小
+
+你可以使用 [antd-dayjs-webpack-plugin](https://github.com/ant-design/antd-dayjs-webpack-plugin) 插件用 Day.js 替换 momentjs 来大幅减小打包大小。
+
+```
```
Comment (afc163) [inline docs/react/use-with-create-react-app.zh-CN.md]: js

File: docs/react/use-with-create-react-app.zh-CN.md [hunk 2]
```diff
@@ -201,6 +201,21 @@ module.exports = override(
...
+const { override, addWebpackPlugin } = require('customize-cra');
+const AntdDayjsWebpackPlugin = require('antd-dayjs-webpack-plugin');
```
Comment (afc163) [inline docs/react/use-with-create-react-app.zh-CN.md]: 缺少 yarn add antd-dayjs-webpack-plugin 的说明

**Review by afc163 — APPROVED** [review]

**Pull request merged**